### DISCUSSION TITLE
Edit webrtc html template

### DISCUSSION THREAD

**Discussion by PrattTechnologies** [bluenviron/mediamtx#3005] [Questions] [answered]
### Question

Hi

Is there a way to edit the html template that the webrtc stream is published in?

Thank you

**Comment by asturl** [comment]
Shouldn't you be able to edit the **publish_index.html** to your liking? As far as I can tell it's all in there.

If you're using the WebRTC streams in your own application, personally I just re-appropriated the connection code (WHEPClient) from the publish site and open the WebRtc connection directly in a video element, with full control over the appearance.

**Comment by aler9** [comment, marked as answer]
Hello, the WebRTC page bundled inside the server can be controlled by using query parameters, in particular:

```
http://localhost:8889/mystream?controls=true&muted=true&autoplay=true&playsinline=true
```

If you need more complex changes, you can copy the `publish_index.html` or `read_index.html` on on a third party HTTP server and change endpoints in order to point to the original MediaMTX instance.

**Reply by PrattTechnologies** [reply]
Thank you!

[a comment by an automated account is not shown]
